forge/forge/tensor.py

```diff
@@ -1339,7 +1339,12 @@ def eval_op(op_type, inputs):
 
 
 def consteval_input(consteval_trace, name: str, inputs: Dict[str, torch.Tensor], is_forge: bool) -> torch.Tensor:
-    return consteval_tensor(consteval_trace, name, inputs, is_forge, "Forward")
+    const_eval_tensor = consteval_tensor(consteval_trace, name, inputs, is_forge, "Forward")
+    # We need to change stride of new tensor explicitly by creating new tensor with shape of const_eval_tensor,
```

Comment:
Can we layer this explanation? In sum, to answer the following questions in order:
1. What we're doing?
2. Why we're doing it?
3. Specifics, Notes, Comments, etc. about how we implement it for the compiler.

Explanation is good, but I think it'll be good just to structure it a bit more to be easier to follow :))

Reply:
Updated the comments.